doc/source/technology_showcase_examples/techdemo-25/ex_25-tecstent.rst

```diff
@@ -0,0 +1,63 @@
+
+.. DO NOT EDIT.
+.. THIS FILE WAS AUTOMATICALLY GENERATED BY SPHINX-GALLERY.
+.. TO MAKE CHANGES, EDIT THE SOURCE PYTHON FILE:
+.. "technology_showcase_examples\techdemo-25\examples-ex_25-tecPCB.py"
+.. LINE NUMBERS ARE GIVEN BELOW.
+
+.. only:: html
+
+    .. note::
+        :class: sphx-glr-download-link-note
+
+        Click :ref:`here <sphx_glr_download_examples_examples_technology_showcase_examples-ex_25-tecPCB.py>`
+        to download the full example code
+
+.. rst-class:: sphx-glr-example-title
+
+.. _sphx_glr_download_examples_examples_technology_showcase_examples-ex_25-tecPCB.py:
```

Comment:
```suggestion
        Click :ref:`here <sphx_glr_download_examples_examples_technology_showcase_examples-ex_25-tecPCB.py>`
        to download the full example code

.. rst-class:: sphx-glr-example-title

.. _sphx_glr_download_examples_examples_technology_showcase_examples-ex_25-tecPCB.py:
```

I'm not sure if this link ``sphx_glr_download_examples_examples_technology_showcase_examples-ex_25-tecPCB.py`` is going to look broken.

Reply:
Because it says "to download the full example code"... we should probably link here the python file instead.

Indeed in the tech demo 20 is also wrong.

Reply:
You're right, I'll make the changes